python/tk_multi_publish2/publish_tree_widget/publish_tree_widget.py

```diff
@@ -65,6 +65,12 @@ def __init__(self, parent):
 
         # forward double clicks on items to the items themselves
         self.itemDoubleClicked.connect(lambda i, c: i.double_clicked(c))
+        
+        # workaround to make the scrollbar work properly for QT versions < 5 on macOS
+        if QtCore.__version__.startswith("4."):
+            import sys
```

Comment:
all imports should happen at the top of the file!

Reply:
I saw it in other places and I figured that it saved importing the file when not needed.
I'll move it over.